release-notes/docs/3.7.0.md

```diff
@@ -0,0 +1,180 @@
+# 3.7.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Configurable hierarchies](#configurable-hierarchies)
+- [Complete reference configuration](#complete-reference-configuration)
+- [Optimised form HTML generation](#optimised-form-html-generation)
+- [Purging server-side](#purging-server-side)
+- [Warn when replicating too many docs](#warn-when-replicating-too-many-docs)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+A new OS level dependency (`xsltproc`) is required for generating XFORM `html` on the server. 
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## Configurable hierarchies
+
+It is now possible to configure contact hierarchies.
+This allows for better tailored solutions for new projects and opens up new configuration 
+possibilities: for example having multiple person types or unbalanced place trees. 
+The previously hardcoded contact hierarchy has now become the default hierarchy and 
+existing projects will continue using it. For more information, please read the
+[configuration information](https://github.com/medic/medic-docs/blob/master/configuration/app-settings.md#configuring-the-contact-hierarchy).
+
+## Complete reference configuration
+
+Version 3.7.0 includes a complete reference configuration that is enabled automatically 
+on fresh application installations. It provides a foundation for forms, data fields, 
+and analytics and has been designed to model an ideal way of structuring and organizing 
+a workflow, its configuration code, and testing framework.
+
+## Optimised form HTML generation
+
+Generating XFORM `html` and `xml` is moved from being executed on every device to 
```

Comment:
Conceptually this only changes the HTML generation, not the XML (which is done when converting from the XLSForm in medic-conf)

```suggestion
Generating the HTML for XForms is moved from being executed on every device to 
```